benchmarking/smartseq2/merge_expression_matrix/merge_table_quants.py

```diff
@@ -0,0 +1,196 @@
+import argparse
+import pandas as pd
+import numpy as np
+from google.cloud import storage
+import json
+from os.path import basename
+import sys
+import requests
+
+
+def retrieve_workflow_outputs(uuid, run_name):
+    # load cromwell credential
+    logins = json.load(open('/usr/secrets/broad-dsde-mint-dev-cromwell.json'))
+    # meta_url
```

Comment:
This comment doesn't add anything beyond the name of the `metadata_url`, can be removed or changed to be something like `# URL of cromwell metadata for provided workflow`